com.github.cubitect.cubiomes-viewer.metainfo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+
+  <id>com.github.cubitect.cubiomes-viewer</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+
+  <developer_name>Cubitect</developer_name>
+  <name>Cubiomes Viewer</name>
+  <summary>An efficient graphical Minecraft seed finder and map viewer.</summary>
+  <description>
+    <p>Cubiomes Viewer offers highly customizable seed-finding utilities and a map viewer for the Minecraft biome and structure generation.</p>
+  </description>
+
+  <releases>
+    <release version="2.3.1" date="2022-08-08">
+    </release>
+  </releases>
+
+  <launchable type="desktop-id">com.github.cubitect.cubiomes-viewer.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">etc/screenshot_maingui.png</image>
```

Comment:
NO. This MUST be an absolute URL. The metainfo file is used in places where there is not even the concept of source code.

All you need to do here is to put the proper URL to the image. Which Github will provide to you on the "download" button that is on the page you were linking to.

Reply:
Sorry, your previous reply hadn't loaded for me until I manually refreshed the page for some reason.